code-challenges.test.js

```diff
@@ -61,11 +123,37 @@ var nums1 = [3, 56, 90, -8, 0, 23, 6]
 var nums2 = [109, 5, 9, -59, 8, 24]
 // Expected output: [-59, 109]
 
+describe("When findMinMax is called", () => {
+  it("return [-8,90] if [3, 56, 90, -8, 0, 23, 6] is called",() => {
+    //arrange
+    const argument = nums1
+    //act
+    const actualResult = findMinMax(argument)
+    //assert
+  expect(actualResult).toEqual([-8, 90])
+  })
+  it("return [-59, 109] if [109, 5, 9, -59, 8, 24] is called",() => {
+    //arrange
+    const argument = nums2
+    //act
+    const actualResult = findMinMax(argument)
+    //assert
+  expect(actualResult).toEqual([-59, 109])
+  })
+})
 
+// b) Create the function that makes the test pass.
 
+//pseudocode
+//create a function that takes an array as an argument
+//use .sort to sort array from least to greatest
+//once sorted use .slice to remove first and last numbers in array and .concat them into new array
+//return new array
 
-
-// b) Create the function that makes the test pass.
+const findMinMax = (array) => {
+  let sortedArray = array.sort((a,b) => a-b)
+  return sortedArray.slice(0,1).concat(sortedArray.slice(-1))
```

Comment:
interesting chaining! it would also be possible to initialize an array with specific values